commonknowledge/mapped
# Various copy changes

## Description
This PR includes
- replacing CRM in the UI with the specific CRM type name
- Generating a link to the user's Action Network API keys from their inputted group slug
- Standardising and making UI labels and explanation text clearer

## Motivation and Context
Addresses issues [MAP-313,](https://linear.app/commonknowledge/issue/MAP-313/text-changes-to-the-choose-platform-step-for-action-network), [MAP-436](https://linear.app/commonknowledge/issue/MAP-436/we-shouldnt-use-the-phrase-crm-in-the-user-interface) and [MAP-316](https://linear.app/commonknowledge/issue/MAP-316/improve-consistency-in-ui-labels-names)

## How Can It Be Tested?
Download the branch and run locally
Run through the on-boarding process for as many CRM types as you like but make sure you include Action Network
Observe the new copy and UI labels
Observe that we generate for the user a URL to their Action Network API keys

## How Will This Be Deployed?
Normal CD process

## Types of changes
